src/python/knnPerfTest.py

```diff
@@ -128,7 +128,8 @@ def run_knn_benchmark(checkout, values):
             # '-forceMerge',
             '-quiet']
         print(f'  cmd: {this_cmd}')
+        print("recall\tlatency\tnDoc\tfanout\tmaxConn\tbeamWidth\tvisited\tindex ms")
```

Comment:
My original idea was to print the header once followed by  the rows with values for each test condition